src/gatsby/src/templates/parkSubPage.js

```diff
@@ -65,6 +44,24 @@ export default function ParkSubPage({ data }) {
     useRef(null),
   ]
 
+  let pageSections = []
+  if (hasSections) {
+    let sectionIndex = 0
+    for (const s of sections) {
+      sectionIndex += 1
+      s.sectionIndex = sectionIndex
+      // each section needs an index to be used for in-page navigation
+      // and scrollspy highlighting
+      const titleId = slugify(s.sectionTitle).toLowerCase()
+      pageSections.push({
+        sectionIndex: sectionIndex,
+        display: s.sectionTitle,
+        link: "#" + titleId,
+        visible: true
+      })
+    }
+  }
```

Comment:
The side menu item doesn't get highlighted when I scroll after removing this part 🙃 I'm taking a look at it but not sure why ...
```
      sectionIndex += 1
      s.sectionIndex = sectionIndex
```

Reply:
If the sectionIndex needs to start at 1, then you might need to do `sectionIndex: sectionIndex + 1`
In an array .map it gives the array index, which starts at 0

Reply:
Yea I did the exactly same but it didn't work out for some reason 😅 Maybe it relates to useScrollSpy thing

Reply:
oh, I didn't notice that it was mutating the `sections` variable in there
```js
s.sectionIndex = sectionIndex
```

If that needs to happen then we can leave the code the way it is. It might be good to avoid it if we can, though. Mutating the array members by reference can be hard to keep track of. (And I missed it this morning! 😆)